bgpd/bgp_packet.h

```diff
@@ -48,6 +48,14 @@ DECLARE_HOOK(bgp_packet_send,
 #define ORF_COMMON_PART_PERMIT     0x00
 #define ORF_COMMON_PART_DENY       0x20
 
+#define BGP_UPDATE_EOR_PKT(_peer, _afi, _safi, _s)       \
```

Comment:
These a standard practice to pass variables in a macro, so that the variables passed never conflicts with any variables from where this macro is being called.